### PR TITLE
Salt day 10

### PR DESCRIPTION
**Pull request by Parzivus**
Part of Day 10. Not too many changes since these files have been pretty heavily edited already.

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 1]
```diff
@@ -99,7 +99,7 @@ My mind is so blank it's as though my brain has turned into tofu.
 [sha:932c1ea514240741390bb70b22bbde61b4fe0fc5]{
 -- Page 6, Offset 24278.
 -- 口は呼吸さえ忘れて固まっている。
-My mouth forgets how to even breathe and seizes up.
+Even my mouth forgets how to breathe and seizes up.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:102]: Prefer original, this form reads a bit more like there's other things that are also forgetting how to breathe

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 2]
```diff
@@ -211,7 +211,7 @@ Even without a full handle on the situation, I can, at the very least, deduce th
 [sha:6f515173681cd1965a406b07f5daaed0e3471341]{
 -- Page 14, Offset 24300.
 -- 「と、とにかくどいてくれ……！
-"Fo-For now just get off me...!
+"Fo-For now, just get off me...!
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:214]: Not sure I'd pause here

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 3]
```diff
@@ -227,7 +227,8 @@ Entering a person's room as you please and straddling them while they're asleep,
 [sha:8636506e61a97ce9b587252888fb651a82420a46]{
 -- Page 15, Offset 24303.
 -- 人を待たせておいて自分はずっと眠っているなんて、いったいどういうつもりなの？」
-Where do you get off, keeping people waiting while you yourself are happily snoozing away?"
+// Not sure what the original line was going for, exactly
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:230]: "Where do you get off" = how dare you. I'd revert.

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 4]
```diff
@@ -383,13 +384,13 @@ You were as still as the grave,
 [sha:239d2775de4a38db2840555aece852bb35d131c5]{
 -- Page 25, Offset 24329.
 -- 「……はあ。不安だったら起こしてくれればよかったのに。
-"...Sigh. If you were that worried, you should have just woken me up.
+"...Hah. If you were that worried, you should have just woken me up.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:387]: In this house we use Sigh

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 5]
```diff
@@ -407,8 +408,9 @@ Though now it's my turn to be anxious,
 [sha:82a726e050cd484a374a7c91c231f8ba16c88643]{
 -- Page 26, Offset 24333.
 -- そうしていたらね、あれだけ湧き上がっていた気持ちも収まって、その後に志貴が目を覚ましたの」
+// Last sentence reads weirdly with "After I'd calmed down" as a subject"
 I found that doing so was the only thing that could soothe those boiling emotions I had felt.
- It was only after I'd calmed down that you woke up."
+ You woke up after I'd calmed down."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:413]: Not sure I agree tbh

File: script/Arcueid/Day 10/10_00_ARC10_3_2.txt [hunk 6]
```diff
@@ -798,13 +799,13 @@ I dutifully cast my eyes around the rooftop.
...
 -- ……確かに。ビルの給水塔の近くに、風を避けるようにテントらしきものが張られていた。
-...Well what do you know.
+...Well, what do you know.
  A small tent has been set up in the lee of the water tower that sits atop the building.
 }
 [sha:6f6185ed30e743776878582bca7a8e8dd98aa8af]{
 -- Page 221, Offset 24830.
 -- この娘、あのテントにいたのか？
-Was this girl inside that tent?
+Is this girl inside that tent?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3_2.txt:808]: ?? She's standing right in front of him

File: script/Arcueid/Day 10/10_00_ARC10_1.txt (hunk 3 above)
Comment (Parzivus) [reply script/Arcueid/Day 10/10_00_ARC10_1.txt:230]: Maybe it's just me, but I've never heard this term before now. Am I the only one?

File: script/Arcueid/Day 10/10_00_ARC10_3_2.txt (hunk 6 above)
Comment (Parzivus) [reply script/Arcueid/Day 10/10_00_ARC10_3_2.txt:808]: Misread the context on this one with tired brain

File: script/Arcueid/Day 10/10_00_ARC10_1.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_1.txt:230]: I don't think it's a Britishism, check with Ben.

Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_1.txt:230]: commented on discord, but not a britishism. Common expression

**Pull request merged**